packages/react-dev-utils/formatWebpackMessages.js

```diff
@@ -16,8 +16,14 @@ function isLikelyASyntaxError(message) {
 
 // Cleans up webpack error messages.
 function formatMessage(message) {
-  let lines = message.split('\n');
+  let lines = [];
 
+  if (typeof message === 'string' || message instanceof String) {
+    lines = message.split('\n');
+  } else if ('message' in Object.keys(message)) {
```

Comment:
When using against an array, `in` operator expects us to provide the index, not the value itself  (as can be seen on the [mdn examples](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in#examples)), so using it on the object should be enough =)

```suggestion
  } else if ('message' in message) {
```

Reply:
@vagnervst 
Thanks for your comment, but as I see `message` in this line should be an `object` not an `array` (see webpack 5 link in PR description).
So checking for a index shouldn't be the case here...

Reply:
Thanks for your contribution! @femesq

`message` is indeed an object, by array I mean the one returned by `Object.keys`. You can check this out on this example I made on jsbin: https://jsbin.com/midaguripe/edit?js,console

If you want the code directly:
```js
const o = {
  message: 'errored',
}

console.log({
  withKeys: 'message' in Object.keys(o), // evaluates to false
  withoutKeys: 'message' in o, // evaluates to true
})
```

Reply:
Got it!
You are right! Thanks for pointing out!